Library/Formula/mupdf-tools.rb

```diff
@@ -0,0 +1,41 @@
+class MupdfTools < Formula
+  homepage "http://mupdf.com"
+  url "http://mupdf.com/downloads/mupdf-1.6-source.tar.gz"
+  sha1 "491d7a3b131589791c7df6dd8161c6bfe41ce74a"
+
+  depends_on :macos => :snow_leopard
+  depends_on "openssl"
+
+  conflicts_with "homebrew/x11/mupdf",
+    :because => "mupdf-tools and mupdf install the same binaries."
```

Comment:
We don't add conflicts with taps in core but you could submit a PR to add this in `homebrew/x11` after this is merged.